### ISSUE TITLE
ExpW and FER test data

### ISSUE THREAD

**Issue by manogna-s** [HCPLab-SYSU/CD-FER-Benchmark#17]
Dear authors,

ExpW does not have a validation split but it appears that you've used a validation set for evaluation. Could you please provide the val lists you used for FER2013 and ExpW?

Thanks,
Manogna

**Comment by putao537** [comment]
> Dear authors,
> 
> ExpW does not have a validation split but it appears that you've used a validation set for evaluation. Could you please provide the val lists you used for FER2013 and ExpW?
> 
> Thanks, Manogna

Please wait a few days.

**Comment by putao537** [comment]
[FER2013.zip](https://github.com/HCPLab-SYSU/CD-FER-Benchmark/files/7852861/FER2013.zip)

ExpW has no standard split. In AGRA, we just randomly divide the dataset into three subsets (train, val, test), and ensure that the number of samples in each subset is as consistent as possible for each category.

Feel free to reopen this issue, if you still have any questions.

**Comment by manogna-s** [comment]
Thanks a lot for sharing the FER2013 splits!

It would be great if you could also share the ExpW splits you used so that we have the same reference. 
Also, can you confirm if the results reported in AGRA paper are the validation split results(based on the code).  If so, is the test split not used at all. Can you please confirm on the evaluation protocol for these datasets?

Thanks and regards,
Manogna

**Comment by putao537** [comment]
> Thanks a lot for sharing the FER2013 splits!
> 
> It would be great if you could also share the ExpW splits you used so that we have the same reference. Also, can you confirm if the results reported in AGRA paper are the validation split results(based on the code). If so, is the test split not used at all. Can you please confirm on the evaluation protocol for these datasets?
> 
> Thanks and regards, Manogna

The ExpW split used in AGRA lost. I only have the code which can generate random split list.

In the AGRA, the report results are the validation split results unless if there only have  train set and test set.

**Comment by manogna-s** [comment]
Thanks a lot for such quick response. 

Could you share the code you have to generate the splits? That way atleast the approx no. of images in each split can be maintained by us. 
Also, by any chance if you have ExpW dataset on any drive, it would be immensely helpful if you could share the link. The files we downloaded seems to be corrupted and we are not able to find any alternative links to download the data. 

Thanks and regards,
Manogna

**Comment by putao537** [comment]
![image](https://user-images.githubusercontent.com/22413761/149170246-4d37d2b5-6dd6-4434-b29f-8397bd50cc9f.png)
Due to the agreement, I can't share ExpW dataset with you.

**Comment by manogna-s** [comment]
Thanks a lot for sharing the code!
